### DISCUSSION TITLE
can I remove these

### DISCUSSION THREAD

**Discussion by m4saurabh** [hrsh7th/nvim-cmp#1562] [Q&A] [answered]
![Screenshot_20230508_121203](https://user-images.githubusercontent.com/79586784/236753966-dff6d5d9-bf2e-4eec-9051-00c71f24a59a.png)

**Comment by bew** [comment]
This white text column? Or these 3 lines specifically?

**Comment by hrsh7th** [comment, marked as answer]
You can remove it with the following.

```lua
cmp.setup {
  formatting = {
    fields = { 'abbr', 'kind', 'menu', },
  }
}
```

Remove `menu` field.

You can see docs.
